### PR TITLE
Minimized outdated content

### LINKED ISSUE #207: Minimize outdated comments
It would be nice if outdated Build comments were minimized when outdated (visually better).

There is a mutation in the GraphQL API that can be called to perform that: https://docs.github.com/en/graphql/reference/mutations#minimizecomment

See [Managing disruptive comments](https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments) for more info.

### PR DESCRIPTION
**Pull request by gastaldi**
- Fixes #207

### CODE REVIEW

File: src/main/java/io/quarkus/bot/HideOutdatedWorkflowRunResults.java [hunk 1]
```diff
@@ -29,10 +29,12 @@
...
     private static final Logger LOG = Logger.getLogger(HideOutdatedWorkflowRunResults.class);
 
-    private static final String HIDE_MESSAGE_PREFIX = "_This workflow status is outdated as a new workflow run has been triggered._\n"
-            + "\n"
-            + "<details>\n"
-            + "\n";
+    private static final String HIDE_MESSAGE_PREFIX = """
+            _This workflow status is outdated as a new workflow run has been triggered._
+
+            <details>
```
Comment (gsmet) [inline src/main/java/io/quarkus/bot/HideOutdatedWorkflowRunResults.java]: Wondering if we should remove the `<details>` thing if the minimization works well?

Comment (gastaldi) [reply src/main/java/io/quarkus/bot/HideOutdatedWorkflowRunResults.java]: I thought about the same, but I wanted to keep at least the Hide message prefix to explain why the comment was hidden

Comment (gsmet) [reply src/main/java/io/quarkus/bot/HideOutdatedWorkflowRunResults.java]: Yeah I totally agree we should keep the sentence. I'll do a few tests.

**Comment by gsmet** [comment]
Not sure exactly what's going on but, trying this, I ended up with:
```
Exception: java.util.concurrent.CompletionException: java.lang.Error: Unresolved compilation problem: 

	at io.quarkus.arc.impl.EventImpl.handleExceptions(EventImpl.java:205)
	at io.quarkus.arc.impl.EventImpl$1.get(EventImpl.java:102)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$13.runWith(VertxCoreRecorder.java:543)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecueadBody.run(EnhancedQueueExecutor.java:1478)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.Error: Unresolved compilation problem: 

	at io.quarkus.bot.HideOutdatedWorkflowRunResults.hideOutdatedWorkflowRunResults(HideOutdatedWorkflowRunResults.java:71)
	at io.quarkus.bot.AnalyzeWorkflowRunResults.analyzeWorkflowResults(AnalyzeWorkflowRunResults.java:106)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass.analyzeWorkflowResults$$superforward1(Unknown Source)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass$$function$$7.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.proceed(InvocationInterceptor.java:62)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.monitor(InvocationInterceptor.java:49)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass.analyzeWorkflowResults(Unknown Source)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer.analyzeWorkflowResults_5faf52dd037aecb35cbd62097faa01b3382161b8(Unknown Source)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass.analyzeWorkflowResults_5faf52dd037aecb35cbd62097faa01b3382161b8$$superforward1(Unknown Source)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass$$function$$8.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.proceed(InvocationInterceptor.java:62)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.monitor(InvocationInterceptor.java:49)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Subclass.analyzeWorkflowResults_5faf52dd037aecb35cbd62097faa01b3382161b8(Unknown Source)
	at io.quarkus.bot.AnalyzeWorkflowRunResults_Multiplexer_Observer_analyzeWorkflowResults_5faf52dd037aecb35cbd62097faa01b3382161b8_75bb4d3fa7668859bcd3a39279d6eb483f9be4fc.notify(Unknown Source)
	at io.quarkus.arc.impl.EventImpl$Notifier.notifyObservers(EventImpl.java:320)
	at io.quarkus.arc.impl.EventImpl$Notifier.notify(EventImpl.java:302)
	at io.quarkus.arc.impl.EventImpl$1.get(EventImpl.java:101)
	... 8 more
```
with JDK 17.

**Comment by gsmet** [comment]
Trying again, it might have been my Eclipse triggering a JDK 11 build in the background.

**Comment by gastaldi** [comment]
Heh, loved the 🌘

**Comment by gastaldi** [comment]
---
> :waning_crescent_moon: **_This workflow status is outdated as a new workflow run has been triggered._**
---

**Comment by gsmet** [comment]
Yeah, I couldn't find anything better but open to suggestion :).

**Comment by gsmet** [comment]
Thanks!

Looks like adding the GraphQL support was worth it :).

**Comment by gastaldi** [comment]
> Looks like adding the GraphQL support was worth it :).

Yeap, that allows us to use features that are not (and probably never will be) available in the REST API

**Pull request merged**